eva-server/src/main/java/uk/ac/ebi/eva/server/repository/VariantEntityRepository.java

```diff
@@ -1,18 +1,39 @@
+/*
+ * European Variation Archive (EVA) - Open-access database of all types of genetic
+ * variation data from all species
+ *
+ * Copyright 2015 EMBL - European Bioinformatics Institute
```

Comment:
This and the rest of new copyright declarations should belong to 2016 since that is the year when the files were created.